script/Arcueid/Day 5/05_00_ARC05_9_2.txt

```diff
@@ -7,52 +7,53 @@ It has to be gliding.
 [sha:eefe338a71ab6b56ff5e167521f134a9cd7de89c]{
 -- Page 10021, Offset 10384.
 -- いや、正しくは落下に近い。
-No, more accurately closer to falling.
+Or rather, something more akin to falling.
 }
 [sha:ec152f17a92ded8286eb9b792d3ca1f19305b1b4]{
 -- Page 10021, Offset 10385.
 -- ……だができるか？　たどり着けるかは運任せだ。
-But can I do it...? It's up to fate whether or not I make it.
+But can I do it? Whether I make it or not would be up to chance.
 }
 [sha:64ee43d68cb9b8c3c9f23d30f317a561def29607]{
 -- Page 644, Offset 10386.
 -- 間違いなく無事では済まない。何もかも上手くいったとしても、着地の衝撃で<筋|に><肉|く>はえぐれ<骨|ほ><格|ね>は折れる。
 It's not %{i}safe%{/i} by any stretch of the imagination.
  Even if everything goes well,
- the impact of the landing will tear my muscles and break my bones.
+ the impact of the landing will tear my muscles apart, or even break my bones.
 }
 [sha:3354128dbc527435548c5e260ba24b625c3c3a19]{
 -- Page 644, Offset 10387.
 -- おまえは、その痛みに耐えられるか？
-Will you really be able to endure that pain?
+//Pronoun changes like this can be hard to parse in English, as they're more fundamental.
+Will I really be able to endure that pain?
 }
 [sha:86a2b6a5fa384f9e6dfd4280bd94808aad9643ec]{
 -- Page 645, Offset 10388.
 -- 俺の迷いを突くかのような弾丸を寸前で止める。
-// B: 俺の迷いを突くかのよう is some nonsense
```

Comment:
stop deleting my comments. i am no longer asking.
![image](https://user-images.githubusercontent.com/16092757/198722739-7add489d-576e-42c2-8aaa-6054dfc1846a.png)

Reply:
You should have made it funnier if you wanted it to live